xc/device.ml

```diff
@@ -2804,7 +2810,8 @@ module Backend = struct
           let ifname          = sprintf "tap%d.%d" domid devid in
           let uuid, _  as tap = tap_open ifname in
           let args =
-            [ "-device"; sprintf "%s,netdev=tapnet%d,mac=%s,addr=%x" nic_type devid mac (Config.NIC.addr ~devid ~index)
+            [ "-device"; sprintf "%s,netdev=tapnet%d,mac=%s,addr=%x%s" nic_type devid mac (Config.NIC.addr ~devid ~index)
+                (String.concat "," ("" :: Config.NIC.extra_flags))
```

Comment:
Could the `String.concat` be avoided by adding using `"...,addr=%x%s,"`? It is possible that `Config.NIC.extra_flags` is empty?

Reply:
Yes extra_flags is empty for qemu-upstream-compat, I'll find another way to create this string.